mods/d2k/languages/lua/en.ftl

```diff
@@ -45,10 +45,12 @@ eliminate-ordos-units-reinforcements = Eliminate all Ordos units and reinforceme
 destroy-harkonnen = Destroy the Harkonnen.
 
 ## atreides-04
+fremen-leader = Fremen Leader
 sietch-integrity = Sietch structural integrity: { $integrity }%
 protect-fremen-sietch = Protect the Fremen Sietch.
 keep-sietch-intact = Keep the Sietch { $integrity }% intact!
 fremen-sietch-southeast = Fremen Sietch detected to the southeast.
+harkonnen-units-approaching = Harkonnen units approaching!
```

Comment:
the added d2k translations are unused

Reply:
I don't know how I managed to test the script and then forget to push it. That _should_ be fixed now.